scripts/index-services.py

```diff
@@ -2,11 +2,11 @@
 """Read services from the API endpoint and write to search-api for indexing.
 
 Usage:
-    index-services.py <stage> --api-token=<api_access_token> --search-api-token=<search_api_access_token> [options]
+    index-services.py <stage> --frameworks=<frameworks> --api-token=<api_access_token> --search-api-token=<search_api_access_token> [options]
```

Comment:
To appease PEP8 you can drop the later command line args in the "usage" on to the next line and it works OK, see e.g. https://github.com/alphagov/digitalmarketplace-scripts/blob/master/scripts/notify-suppliers-whether-application-made-for-framework.py#L6